.gitignore

```diff
@@ -1,7 +1,6 @@
 # Python cache
 __pycache__
 *.pyc
-build
```

Comment:
xla customcall marker has a build folder, but should be ignored

Reply:
yeah I fixed it in #367